ah/kubernetes_clusters.go

```diff
@@ -22,126 +22,124 @@ import (
 	"net/http"
 )
 
-// Cluster object
-type Cluster struct {
-	ID           string `json:"id,omitempty"`
-	Name         string `json:"name,omitempty"`
-	DatacenterID string `json:"datacenter_id,omitempty"`
-	State        string `json:"state,omitempty"`
-	Number       string `json:"number"`
-	CreatedAt    string `json:"created_at"`
-	Count        int    `json:"count"`
-	PlanID       int    `json:"plan_id"`
-	Vcpu         int    `json:"vcpu"`
-	Ram          int    `json:"ram"`
-	Disk         int    `json:"disk"`
+// KubernetesCluster object
+type KubernetesCluster struct {
+	ID               string               `json:"id,omitempty"`
+	Name             string               `json:"name,omitempty"`
+	TokenID          string               `json:"token_id,omitempty"`
+	DatacenterID     string               `json:"datacenter_id,omitempty"`
+	DatacenterSlug   string               `json:"datacenter_slug,omitempty"`
+	PrivateNetwork   string               `json:"private_network,omitempty"`
+	State            string               `json:"state,omitempty"`
+	Number           string               `json:"number"`
+	CreatedAt        string               `json:"created_at"`
+	AccountID        string               `json:"account_id"`
+	PrivateNetworkID string               `json:"private_network_id"`
```

Comment:
do we need both private_network and private_network_id? If yes and we assume it should work the same way as datacenters, it would make sense to have it like `PrivateNetworkID` and `PrivateNetworkName`, because `PrivateNetwork` doesn't look obvious. By the way, consider changing it in API Service as well.
And also `PrivateNetworkID` should be `omitempty` as in case when you pass name it's supposed that you won't pass id. And also it would be good to have both fields just one below another as we have for DatacenterID, DatacenterSlug